### PR TITLE
Fixing django3.2 issue with app config.

### PR DESCRIPTION
**Pull request by awais786**
https://openedx.atlassian.net/browse/BOM-2653

Fixing django3.2 tests issue.

### CODE REVIEW

File: openedxstats/apps/slackdata/apps.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
+import django
 from django.apps import AppConfig
 
 
 class SlackdataConfig(AppConfig):
     name = 'slackdata'
-
+    if django.VERSION >= (3, 2):
+        default_app_config = False
```
Comment (awais786) [inline openedxstats/apps/slackdata/apps.py:8]: https://docs.djangoproject.com/en/3.2/ref/applications/#django.apps.AppConfig.default

prevent Django from selecting a configuration class automatically.
it already has defined path in Installed apps.

File: openedxstats/apps/sites/apps.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
+import django
 from django.apps import AppConfig
 
 
 class SitesConfig(AppConfig):
     name = 'sites'
-
+    if django.VERSION >= (3, 2):
```
Comment (awais786) [inline openedxstats/apps/sites/apps.py:7]: https://docs.djangoproject.com/en/3.2/ref/applications/#django.apps.AppConfig.default

prevent Django from selecting a configuration class automatically.
it already has defined path in Installed apps.

Comment (awais786) [reply openedxstats/apps/sites/apps.py:7]: https://github.com/edx/openedxstats/blob/master/openedxstats/settings/base.py#L44

**Review by mraarif — APPROVED** [review]

**Comment by nedbat** [comment]
When you squash these commits, they should have a conventional commit message using "fix:" as the label.

**Review by UsamaSadiq — APPROVED** [review]

**Pull request merged**